examples/stylesheets/styles.css

```diff
@@ -553,6 +553,7 @@ h3.option {
     padding-left: 10px;
     border-top: 5px solid #CCC;
     margin-top: 10px;
+	min-height: 24px;

```

Comment:
bug, if the h3 content is empty, there will be wrong height due to no clear float